packages/jest-cli/src/watch.js

```diff
@@ -321,6 +321,10 @@ export default function watch(
     ).find(plugin => getPluginKey(plugin, globalConfig) === key);
 
     if (matchingWatchPlugin != null) {
+      if (isRunning) {
```

Comment:
This is something that we are already handling correctly for "non plugin" keys https://github.com/facebook/jest/pull/7222/files#diff-d8a3d0a3a8b42ab87b18835ad8f7cb46R309